src/main/kotlin/calculator/ArithmeticExpression.kt

```diff
@@ -0,0 +1,21 @@
+package calculator
+
+private const val DELIMITER = " "
+
+data class ArithmeticExpression(
+    private val expression: String
+) {
+    val operators: List<Operator>
+
+    val operands: List<Double>
+
+    init {
+        require(expression.isNotBlank() && expression.isNotEmpty()) { "연산식은 공백이 될수 없습니다." }
+        val split = expression.split(DELIMITER)
+
+        operators = split.filterIndexed { i, _ -> i % 2 == 1 }
+            .map { Operator.find(it) }
+        operands = split.filterIndexed { i, _ -> i % 2 == 0 }
+            .map { s: String -> s.toDouble() }
```

Comment:
```suggestion
            .map { it.toDouble() }
```
- 여기도 `it`을 활용할 수 있을 거 같아요.
- 만약 Double형이 아닐 경우 특정한 예외를 발생시키고 싶다면 `toDoubleOrNull() ?: throw IllegalArgumentException()` 등 처럼 나타낼 수 있을 것 같습니다 :)